html/index-bulk.html

```diff
@@ -0,0 +1,528 @@
+<script>
+  experimentTable.render({
+    host: 'http://www.ebi.ac.uk/',
+    resource: 'gxa/json/experiments',
+    //link to `${host}/${header.resource}/${data[header.link]}/${header.endpoint}`
+      species: ``,
+     downloadTooltip: `<ul>
+          <li>Raw filtered count matrix after quantification</li>
+          <li>Normalised filtered count matrix after quantification</li>
+          <li>TPM (transcripts per million RNA molecules - only available for SmartSeq)</li>
+          <li>SDRF file with the experimental design metadata</li>
+        </ul>`,
+      tableHeader:
+        [
+            //{title: `index`, width: 60, dataParam: null, link: null},
+            {type: `sort`, title: `Type`, width: 50, dataParam: `experimentType`,
+                image:
+                    {RNASEQ_MRNA_BASELINE:
+                            `<img src="http://localhost:8080/gxa/resources/images/experiments-table/baseline.png" alt="baseline"> ` ,
+                        MICROARRAY_2COLOUR_MRNA_DIFFERENTIAL:
+                            `<img src="http://localhost:8080/gxa/resources/images/experiments-table/differential.png" alt="differential">`,
+                        MICROARRAY_1COLOUR_MRNA_DIFFERENTIAL:
+                            `<img src="http://localhost:8080/gxa/resources/images/experiments-table/differential.png" alt="differential">`,
+                        RNASEQ_MRNA_DIFFERENTIAL:
+                            `<img src="http://localhost:8080/gxa/resources/images/experiments-table/differential.png" alt="differential">`,}},
+          {type: `sort`, title: `Loaded date`, width: 120, dataParam: `lastUpdate`},
+          {type: `search`, title: `species`, width: 200, dataParam: `species`},
+          {type: `search`, title: `experiment description`, width: 400, dataParam: `experimentDescription`,
+            link: `experimentAccession`, resource: `experiments`, endpoint: `Results`},
+          {type: `sort`, title: `Number of assays`, width: 120, dataParam: `numberOfAssays`,
+            link: `experimentAccession`, resource: `experiments`, endpoint: `Experiment%20Design`},
+            {type: `sort`, title: `Comparisons`, width: 100, dataParam: `numberOfContrasts`,
+                link: `experimentAccession`, resource: `experiments`, endpoint: `Experiment%20Design`},
+            {type: `search`, title: `experiment factors`, width: 200, dataParam: `experimentalFactors`},
+            {type: `search`, title: `array designs`, width: 220, dataParam: `arrayDesignNames`,
+                link: `arrayDesigns`, resource: `arrayexpress/arrays`, endpoint: ``},
+            {type: ``, title: `ArrayExpress`, width: 100, dataParam: `numberOfContrasts`,
+                image: `<span class="icon icon-generic" data-icon="L" title="View in Array Express"></span>`,
+                link: `experimentAccession`, resource: `arrayexpress/experiments`, endpoint: ``},
```

Comment:
Not all experiments are deposited in ArrayExpress. Remember that we added a feature whereby links to ArrayExpress in the experiment page would be polled first to know if the experiment exists in AE, and if not, we wouldn’t display it.

This column should be something like *Links to other repositories* or *Resource links*, since experiments will have links to different services/projects (ArrayExpress, PRIDE, HCA, Encode, ENA, etc.), and the back end will be reponsible for including such links in the JSON payload. For the time being, though, let’s leave this column out.